renku/ui/cli/config.py

```diff
@@ -97,13 +97,19 @@
 | ``lfs_threshold``              | Threshold file size below which     | ``100kb`` |
 |                                | files are not added to git LFS      |           |
 +--------------------------------+-------------------------------------+-----------+
-| ``show_lfs_message``           | Whether to show messages about      | ``True``  |
+| ``show_lfs_message``           | Whether to show messages about      | ``true``  |
```

Comment:
We either need to change `True` to `true` whenever we use this config (with a `.lower()` to be compatible with older projects) or just undo this change here.